framework/include/utils/MooseUtils.h

```diff
@@ -778,6 +779,15 @@ struct IsLikeReal<DualReal>
   static constexpr bool value = true;
 };
 
+/**
+ * Construct a valid bounding box from 2 arbitrary points
```

Comment:
Can you you put a comment in here as to _why_ this is the preferred method to build a bounding box instead of using the standard constructor?

If you can think of another method name that would suggest that this picks the correct orientation, that would be great too. But, I'm struggling to come up with one so it's not necessary.

Reply:
> Can you you put a comment in here as to why this is the preferred method to build a bounding box instead of using the standard constructor?

Yeah, I was thinking about that. But, not all bounding boxes in the framework are/should be constructed this way. So, I will add _when_ and also _why_ to use this 👍 

> If you can think of another method name that would suggest that this picks the correct orientation, that would be great too. But, I'm struggling to come up with one so it's not necessary.

I don't think the name should suggest anything about the orientation at all. It is an implementation detail of the `BoundingBox` class. You as a user want a bounding box from 2 points. Why do you need to care about how it is done inside the class, right? The explaining comment should be enough here...